### PR TITLE
Docs: Document how to pass in a custom JChannel to the transport conf…

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

**Comment by galderz** [comment]
That's an interesting JIRA number ;)

File: documentation/src/main/asciidoc/user_guide/configuration.adoc [hunk 1]
```diff
@@ -252,6 +253,22 @@ GlobalConfiguration globalConfig = new GlobalConfigurationBuilder().transport()
...
+If you want to construct the JGroups link:http://www.jgroups.org/manual4/index.html#JChannel[JChannel] by yourself, you can do so.
+
+NOTE: The JChannel must not be already connected.
+
+[source,java]
+----
+GlobalConfigurationBuilder global = new GlobalConfigurationBuilder();
+JChannel jchannel = new JChannel();
+// Configure the jchannel to your needs.
+JGroupsTransport transport = new JGroupsTransport(jchannel);
```
Comment (pruivo) [inline documentation/src/main/asciidoc/user_guide/configuration.adoc]: we have a JGroupsChannelLookup, but I guess it isn't easier to use when compared with your example :) https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsChannelLookup.java

**Comment by tristantarrant** [comment]
JGroupsChannelLookup is far too complex for most user's needs, as it is tailored for WildFly which needs to control the lifecycle of the channel too.

**Comment by wburns** [comment]
Maybe just add a JIRA and we can integrate.

**Comment by pruivo** [comment]
is this PR still valid? It needs rebase :)

**Comment by ryanemerson** [comment]
@tristantarrant Can you rebase and add a jira number so that it makes 9.4.0.Final?

**Comment by tristantarrant** [comment]
Closed in favor of #6314

**Pull request closed without merging**